Kubeflow getting panic: runtime error: invalid memory address or nil pointer dereference when kfctl.sh generate k8s

<!-- This form is for bug reports and feature requests ONLY!
-->

**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
> /kind bug
>
> /kind feature


**What happened**:

Running kubeflow kftctl.sh generate k8s crashes ksonnet with panic: runtime error: invalid memory address or nil pointer dereference.

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1057c42]

goroutine 1 [running]:
github.com/ksonnet/ksonnet/vendor/github.com/ksonnet/ksonnet-lib/ksonnet-gen/ksonnet.parsePaths(0xc4204cc400, 0x6, 0x1, 0xc)
	/Users/oren.shomron/go/src/github.com/ksonnet/ksonnet/vendor/github.com/ksonnet/ksonnet-lib/ksonnet-gen/ksonnet/paths.go:33 +0x1f1
github.com/ksonnet/ksonnet/vendor/github.com/ksonnet/ksonnet-lib/ksonnet-gen/ksonnet.NewCatalog(0xc4204cc400, 0xc420a56c30, 0x1, 0x1, 0x40, 0x0, 0x0)
	/Users/oren.shomron/go/src/github.com/ksonnet/ksonnet/vendor/github.com/ksonnet/ksonnet-lib/ksonnet-gen/ksonnet/catalog.go:76 +0x1de
github.com/ksonnet/ksonnet/vendor/github.com/ksonnet/ksonnet-lib/ksonnet-gen/ksonnet.GenerateLib(0xc420042180, 0x3a, 0x0, 0x0, 0x0)
	/Users/oren.shomron/go/src/github.com/ksonnet/ksonnet/vendor/github.com/ksonnet/ksonnet-lib/ksonnet-gen/ksonnet/ksonnet.go:25 +0xce
github.com/ksonnet/ksonnet/pkg/util/kslib.astBuilder(0xc420042180, 0x3a, 0xc42299c000, 0x3f0dc0, 0x3ffe00, 0x0, 0x0, 0x0)
	/Users/oren.shomron/go/src/github.com/ksonnet/ksonnet/pkg/util/kslib/ksonnet.go:87 +0x40
github.com/ksonnet/ksonnet/pkg/util/kslib.Ksonnet(0xc42299c000, 0x3f0dc0, 0x3ffe00, 0x0, 0x0, 0x0)
	/Users/oren.shomron/go/src/github.com/ksonnet/ksonnet/pkg/util/kslib/ksonnet.go:83 +0x3fd
github.com/ksonnet/ksonnet/pkg/lib.(*defaultKsLibGenerator).Generate(0xc42218d9e0, 0xc421d99c80, 0xf, 0xc42221593a)
	/Users/oren.shomron/go/src/github.com/ksonnet/ksonnet/pkg/lib/lib.go:62 +0x8e
github.com/ksonnet/ksonnet/pkg/lib.(*Manager).GenerateLibData(0xc422180500, 0xf, 0x2b1e380)
	/Users/oren.shomron/go/src/github.com/ksonnet/ksonnet/pkg/lib/lib.go:106 +0x4c
github.com/ksonnet/ksonnet/pkg/app.ksLibUpdater.UpdateKSLib(0x2b1e380, 0x3942898, 0xc421d9f530, 0xc421d99c80, 0xf, 0xc421d8ae40, 0x2b, 0xc42153b0b0, 0x27, 0x25a9da5, ...)
	/Users/oren.shomron/go/src/github.com/ksonnet/ksonnet/pkg/app/lib_updater.go:44 +0x99
github.com/ksonnet/ksonnet/pkg/app.(*baseApp).AddEnvironment(0xc421da4ee0, 0xc4222245a0, 0xc421d99c80, 0xf, 0x3620200, 0x3, 0x3)
	/Users/oren.shomron/go/src/github.com/ksonnet/ksonnet/pkg/app/base_app.go:532 +0x4be
github.com/ksonnet/ksonnet/pkg/env.(*creator).Create(0xc42222a000, 0xc42222a000, 0xc421da4ee0)
	/Users/oren.shomron/go/src/github.com/ksonnet/ksonnet/pkg/env/create.go:117 +0x9c3
github.com/ksonnet/ksonnet/pkg/env.Create(0x2b26400, 0xc421da4ee0, 0xc421d7c6e0, 0x16, 0x25ae36e, 0x7, 0x25ae36e, 0x7, 0xc421d99c80, 0xf, ...)
	/Users/oren.shomron/go/src/github.com/ksonnet/ksonnet/pkg/env/create.go:42 +0x157
github.com/ksonnet/ksonnet/pkg/appinit.(*initApp).Run(0xc420a578d8, 0x35, 0x47028b0)
	/Users/oren.shomron/go/src/github.com/ksonnet/ksonnet/pkg/appinit/init.go:91 +0x459
github.com/ksonnet/ksonnet/pkg/appinit.Init(0x2b1e380, 0x3942898, 0xc421d9f530, 0x7ffeefbffae0, 0x6, 0xc42153b0b0, 0x27, 0x0, 0x0, 0xc421d99c80, ...)
	/Users/oren.shomron/go/src/github.com/ksonnet/ksonnet/pkg/appinit/init.go:34 +0x155
github.com/ksonnet/ksonnet/pkg/actions.(*Init).Run(0xc421d32140, 0xc421d32140, 0x0)
	/Users/oren.shomron/go/src/github.com/ksonnet/ksonnet/pkg/actions/init.go:113 +0x140
github.com/ksonnet/ksonnet/pkg/actions.RunInit(0xc421d9f500, 0xc421c2e240, 0xe)
	/Users/oren.shomron/go/src/github.com/ksonnet/ksonnet/pkg/actions/init.go:40 +0x5c
github.com/ksonnet/ksonnet/pkg/clicmd.runAction(0xe, 0xc421d9f500, 0x25c00fa, 0x12)
	/Users/oren.shomron/go/src/github.com/ksonnet/ksonnet/pkg/clicmd/actions.go:118 +0x66
github.com/ksonnet/ksonnet/pkg/clicmd.newInitCmd.func1(0xc421ce0280, 0xc421cb8740, 0x1, 0x1, 0x0, 0x0)
	/Users/oren.shomron/go/src/github.com/ksonnet/ksonnet/pkg/clicmd/init.go:156 +0x7fa
github.com/ksonnet/ksonnet/vendor/github.com/spf13/cobra.(*Command).execute(0xc421ce0280, 0xc421cb8700, 0x1, 0x1, 0xc421ce0280, 0xc421cb8700)
	/Users/oren.shomron/go/src/github.com/ksonnet/ksonnet/vendor/github.com/spf13/cobra/command.go:762 +0x468
github.com/ksonnet/ksonnet/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc421bf3180, 0xc421c7e000, 0xc421bf3680, 0xc421bf3400)
	/Users/oren.shomron/go/src/github.com/ksonnet/ksonnet/vendor/github.com/spf13/cobra/command.go:852 +0x30a
github.com/ksonnet/ksonnet/vendor/github.com/spf13/cobra.(*Command).Execute(0xc421bf3180, 0x3942898, 0xc420046064)
	/Users/oren.shomron/go/src/github.com/ksonnet/ksonnet/vendor/github.com/spf13/cobra/command.go:800 +0x2b
main.main()
	/Users/oren.shomron/go/src/github.com/ksonnet/ksonnet/cmd/ks/main.go:48 +0x148

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:


**Anything else we need to know?**:

**Environment**:
- ksonnet version (use `ks version`):
ksonnet version: 0.13.0
jsonnet version: v0.11.
- Kubernetes version (use `kubectl version`):

--- a/Gopkg.lock
+++ b/Gopkg.lock
@@ -598,7 +598,7 @@
   revision = "ae77be60afb1dcacde03767a8c37337fad28ac14"
 
 [[projects]]
-  digest = "1:940f9581e8a768010b6dada7055bc86bd872f7f64e7affe894f39115956073c2"
+  digest = "1:26096952c000895a6b908d301f07d68b9a39a9ba57bd447f99edf3db9f47c4a9"
   name = "github.com/ksonnet/ksonnet-lib"
   packages = [
     "ksonnet-gen/astext",
@@ -610,8 +610,8 @@
     "ksonnet-gen/printer",
   ]
   pruneopts = "UT"
-  revision = "ed0796f3cb97ebc35ae54f543b1814a7c8dae305"
-  version = "v0.1.11"
+  revision = "d03da231d6c8bd74437b74a1e9e8b966f13dffa2"
+  version = "v0.1.12"
 
 [[projects]]
   digest = "1:c568d7727aa262c32bdf8a3f7db83614f7af0ed661474b24588de635c20024c7"

--- a/Gopkg.toml
+++ b/Gopkg.toml
@@ -52,7 +52,7 @@ required = ["k8s.io/kubernetes/pkg/kubectl/cmd/util"]
 
 [[constraint]]
   name = "github.com/ksonnet/ksonnet-lib"
-  version = "v0.1.11"
+  version = "v0.1.12"
 
 [[constraint]]
   name = "github.com/onsi/ginkgo"

